clients/spark/core/src/main/scala/io/treeverse/clients/GarbageCollector.scala

```diff
@@ -242,18 +237,32 @@ object GarbageCollector {
       System.exit(1)
     }
 
-    val shouldMark = hc.getBoolean(LAKEFS_CONF_GC_DO_MARK, true)
-    val shouldSweep = hc.getBoolean(LAKEFS_CONF_GC_DO_SWEEP, true)
-
     if (!shouldMark && !shouldSweep) {
       Console.out.println("Nothing to do, must specify at least one of mark, sweep. Exiting...")
-      System.exit(0)
-    } else if (!shouldMark && hc.get(LAKEFS_CONF_GC_MARK_ID, "").isEmpty) {
+      System.exit(2)
+    } else if (!shouldMark && hc.get(LAKEFS_CONF_GC_MARK_ID, "").isEmpty) { // Sweep-only mode but no mark ID to sweep
```

Comment:
The function gets 2 args (the booleans) from hc passed in as parameters, but also goes and gets another arg (the mark_id string) from hc.  Can we pick one (function with no parameters, gets them from hc, or function gets all parameter values and does not read hc)?